repos/runtime.common.props

```diff
@@ -72,6 +74,8 @@
 
     <CommonBuildArguments>$(FlagParameterPrefix)arch $(Platform)</CommonBuildArguments>
     <CommonBuildArguments>$(CommonBuildArguments) $(FlagParameterPrefix)configuration $(Configuration)</CommonBuildArguments>
+    <CommonBuildArguments>$(CommonBuildArguments) /p:RuntimeConfiguration=$(Configuration)</CommonBuildArguments>
+    <CommonBuildArguments>$(CommonBuildArguments) /p:BypassFrameworkInstallChecks=true</CommonBuildArguments>
```

Comment:
What's `BypassFrameworkInstallChecks` about?

Reply:
When compiling for a full framework, the SDK's targets file includes a check to make sure the framework is installed - this is never true for us because Linux doesn't have these and we use the ReferenceAssemblies packages to compile for full-fx instead.  That being said, I think I just removed the full fx compile so this probably isn't needed right now, I'll double-check.

Reply:
Yup, removing the full fx TFM obsoleted this.  Removed.